artiq/firmware/ksupport/eh_artiq.rs

```diff
@@ -209,7 +209,7 @@ pub unsafe extern fn raise(exception: *const Exception) -> ! {
                 // https://github.com/rust-lang/rfcs/pull/1719
                 function: "__artiq_raise".as_c_slice(),
                 message:  "too many nested exceptions".as_c_slice(),
-                param:    [0, 0, 0]
+                param:    (0, 0, 0, "")
```

Comment:
If you listened and didn't mess with ``params`` you would not need this hack or the ones below.